Add basic example with sample files

sukritsingh opened the pull request:
Address #49 : Providing a basic example script for running the plumed-openmm plugin in a basic metadynamics simulation.

The script `examples/basic-example/plumed-metad-sim.py` is a single script contains reads in a system, state, and topology and runs a metadynamics simulation using the PLUMED scripting language to define collective variables. A sample output file from PLUMED, `colvar`, is also included.

The subdirectory `basic-example` was created to prevent future confusion if other more advanced examples get added.

sukritsingh commented:
It looks like a couple of the checks haven't run yet for some reason? @raimis or @peastman do you have any feedback on the examples so this can be merged in?

peastman commented:
I'm not sure what's going on.  I just told it to rerun all checks, but I'm not convinced it's really doing anything.

sukritsingh commented:
Looks it it never really ran the checks either again?

sukritsingh commented:
Any updates/luck with running these checks?

roshan2004 commented:
Sorry, but I still could not find such an example script.

sukritsingh commented:
I imagine this PR needs to be merged for the example scripts to be added into the main branch. If you're looking for the example scripts I wrote in this PR you could probably find them fastest in the "files changed" tab?

Separately, it does look like all the checks still never ran again, and still need to be rerun prior to merging?

sukritsingh commented:
No worries at all! Everything is Chaos now anyways. 

I've made the following changes and pushed them in the latest commit: 
1. Merged in any upstream changes that were made after this PR fork
2. Deleted the unused `molecule` object 
3. Removed hardcoding of the `platform` to OpenCL

It should be good to rerun the workflow checks again now, if you get a chance!

peastman commented:
Thanks!

CI is failing, but it's not related to this.  I'll fix it in a separate PR.
